client/blocks/reader-full-post/featured-image.jsx

```diff
@@ -23,12 +29,13 @@ export default class FeaturedImage extends React.Component {
 
 		return (
 			<div className="reader-full-post__featured-image">
-				<img src={ this.state.src } onError={ this.handleImageError } alt="" />
+				<img src={ this.state.src } onError={ this.handleImageError } alt={ this.state.alt } />
```

Comment:
Why not use `this.props.alt` without storing it? This is a class React anti-pattern.

https://reactjs.org/blog/2018/06/07/you-probably-dont-need-derived-state.html#anti-pattern-unconditionally-copying-props-to-state

Reply:
Thank you very much! This is now fixed in https://github.com/Automattic/wp-calypso/pull/55378/commits/f8c9e46a0266859e584255db16a79e711def9cff.